test/unit/evault/modules/Vault/batch.t.sol

```diff
@@ -0,0 +1,345 @@
+        assertEq(collaterals[0], address(eTST));
+        assertApproxEqAbs(collateralValues[0], 0.24e18, 0.001e18);
+        assertApproxEqAbs(liabilityValue, 0.083e18, 0.001e18);
+
+        uint256 balance2 = abi.decode(batchItemsResult[7].result, (uint256));
+        uint256 balance3 = abi.decode(batchItemsResult[8].result, (uint256));
+
+        assertEq(balance2, 0.6e18);
+        assertEq(balance3, 1e18);
+
+        // Do a real one
+        items = new IEVC.BatchItem[](8);
+        items[0] = IEVC.BatchItem({
+            onBehalfOfAccount: user1,
+            targetContract: address(eTST),
+            value: 0,
+            data: abi.encodeWithSelector(eTST.transfer.selector, getSubAccount(user1, 1), 1e18)
+        });
+        items[1] = IEVC.BatchItem({
+            onBehalfOfAccount: getSubAccount(user1, 1),
+            targetContract: address(eTST),
+            value: 0,
+            data: abi.encodeWithSelector(eTST.approve.selector, user1, type(uint256).max)
+        });
+        items[2] = IEVC.BatchItem({
+            onBehalfOfAccount: user1,
+            targetContract: address(eTST),
+            value: 0,
+            data: abi.encodeWithSelector(
+                eTST.transferFrom.selector, getSubAccount(user1, 1), getSubAccount(user1, 2), 0.6e18
+            )
+        });
+        items[3] = IEVC.BatchItem({
+            onBehalfOfAccount: address(0),
+            targetContract: address(evc),
+            value: 0,
+            data: abi.encodeWithSelector(evc.enableCollateral.selector, getSubAccount(user1, 1), address(eTST))
+        });
+        items[4] = IEVC.BatchItem({
+            onBehalfOfAccount: address(0),
+            targetContract: address(evc),
+            value: 0,
+            data: abi.encodeWithSelector(evc.enableController.selector, getSubAccount(user1, 1), address(eTST2))
+        });
+        items[5] = IEVC.BatchItem({
+            onBehalfOfAccount: getSubAccount(user1, 1),
+            targetContract: address(eTST2),
+            value: 0,
+            data: abi.encodeWithSelector(eTST2.borrow.selector, 1e18, user1)
+        });
+        items[6] = IEVC.BatchItem({
+            onBehalfOfAccount: address(0),
+            targetContract: address(evc),
+            value: 0,
+            data: abi.encodeWithSelector(evc.enableController.selector, user1, address(eTST2))
+        });
+        items[7] = IEVC.BatchItem({
+            onBehalfOfAccount: user1,
+            targetContract: address(eTST2),
+            value: 0,
+            data: abi.encodeWithSelector(eTST2.borrow.selector, 1e18, user1)
+        });
+
+        evc.batch(items);
+
+        assertEq(eTST.balanceOf(getSubAccount(user1, 1)), 0.4e18);
+        assertEq(eTST2.debtOf(getSubAccount(user1, 1)), 1e18);
+        assertEq(eTST.balanceOf(getSubAccount(user1, 2)), 0.6e18);
+        assertEq(evc.getCollaterals(getSubAccount(user1, 1))[0], address(eTST));
+    }
+
+    //call to unknown module is permitted
```

Comment:
let's rename 'to unknown contract' here and in test name. Modules is out-dated terminology from v1